### ISSUE TITLE
Convolutional Neural Networks to classify dog images (Image Classification)

### ISSUE THREAD

**Issue by Advita9** [abhisheks008/DL-Simplified#917]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : End-to-end Multi-class Dog Breed Classification </br>
:red_circle: **Aim** : Identifying the breed of a dog gives an image of a dog using Convolutional Neural Networks </br>
:red_circle: **Dataset** : https://www.google.com/url?q=https%3A%2F%2Fwww.kaggle.com%2Fc%2Fdog-breed-identification%2Fdata</br>
:red_circle: **Approach** : Adequately understanding our image data and converting it into Tensors, using various models, starting off with a model from Tensorflow Hub and defining the convolutional layers in a Keras deep learning model, comparing prediction probabilities and evaluating it on a real-world dataset.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Advita Shrivastava
- GitHub Profile Link : https://github.com/Advita9
- Email ID : advitashrivastava09@gmail.com
- Participant ID (if applicable): 
- Approach for this Project : Convolutional Neural Networks
- What is your participant role? (Mention the Open Source program) GirlsScript Summer of Code

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Karm-Dave** [comment]
I would like to work on this issue. Here are my details.
    Full name : Karm Dave
    GitHub Profile Link : https://github.com/Karm-Dave
    Email ID : [karm.dave@gmail.com](mailto:karm.dave@gmail.com)
    Approach for this Project : Convolutional Neural Networks
    What is your participant role? GirlsScript Summer of Code Extd 2024

**Comment by Advita9** [comment]
I would like to take up this issue
Full name : Advita Shrivastava
GitHub Profile Link : https://github.com/Advita9
Email ID : [advitashrivastava09@gmail.com](mailto:advitashrivastava09@gmail.com)
Participant ID (if applicable):
Approach for this Project : Convolutional Neural Networks
What is your participant role? (Mention the Open Source program) GirlsScript Summer of Code

**Comment by sagar27sahu** [comment]
Hi  I’d like to work on the task of building a Convolutional Neural Network (CNN) for classifying dog images. The plan would include:

Collecting a dataset of labeled dog images.
Preprocessing the data with techniques like resizing, normalization, and augmentation.
Building and training a CNN model using TensorFlow or PyTorch.
Evaluating and fine-tuning the model for better accuracy.
Testing the final model on new dog images.
Could you please assign this task to me? Looking forward to contributing!"

**Comment by Advita9** [comment]
Hi, with regards to the issue you commented on for End-to-end Multi-class
Dog Breed Classification , I had already provided my details to contribute
to it in my comment. I think there is some kind of confusion. Would be
great if you could take down your comment!

**Comment by abhisheks008** [comment]
This kind of project is already present in this repository. Hence closing this issue as duplicate.
